app/frontend/src/router/index.js

```diff
@@ -34,6 +34,12 @@ const router = new VueRouter({
           name: 'Submission',
           component: () => import(/* webpackChunkName: "submission" */ '@/views/admin/Submission.vue'),
           props: true
+        },
+        {
+          path: 'submission/:ipcPlanId/status',
```

Comment:
Where are we going to stop non-Inspectors from seeing having access to /status?

Reply:
It's to be determined atm I think. Lucas is proposing we show those paths on navbar. I haven't quite figured if we should also be adding a check on the router before logic as well just yet.

Reply:
cool, just didn't want security lost in the shuffle.